### PR TITLE
Tickets are released!

### PR DESCRIPTION
**Pull request by dynamictulip**

### CODE REVIEW

**Review by slang25 — APPROVED** [review]

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -50,8 +50,10 @@
 	<div class="section">
 		<h1>Latest update</h1>
 		<p>
-			<a href="/schedule">Schedule is announced!!</a> Time for last second tweaks to your talks, test
-			your demos and get ready for the big day!!
+			Registration is now open,
+			<a href="https://www.eventbrite.co.uk/e/ddd-southwest-2024-registration-798955125287">
+				get your tickets here</a
+			> and see you on the 27th!
```
Comment (slang25) [inline src/routes/+page.svelte:56]: I know I approved, but just spotted this, is this a rogue new-line?

**Comment by CarlSargunar** [comment]
Looks good to me

File: src/routes/+page.svelte (hunk 1 above)
Comment (CarlSargunar) [reply src/routes/+page.svelte:56]: yeah - vs code threw one in for me too when I did mine - the autoformatting seems a little weird, but the page still worked fine

Comment (slang25) [reply src/routes/+page.svelte:56]: Cool, in which case, ignore me 😄

**Pull request merged**